beets/ui/commands.py

```diff
@@ -1395,6 +1395,55 @@ def move_func(lib, opts, args):
 default_commands.append(move_cmd)
 
 
+def migrate_func(lib, opts, args):
+    items, _ = _do_query(lib, None, False, False)
+
+    not_found = []
+    for item in items:
+        if not os.path.exists(item.path):
+            not_found.append(item)
+
+    found = []
+    for item in not_found:
+        if os.path.exists(item.destination()):
+            found.append(item)
+
+    plural = not_found != 1
+    log.info(u'{0} item' + (u's' if plural else u'') + u' in the database have'
+            u' bad paths, and {1} will be fixed.', len(not_found), len(found))
+    
+    unchanged = len(not_found) - len(found)
+    if unchanged:
+        log.info(u'{0} must be migrated manually.', unchanged)
```

Comment:
For both this log message and the one above, a terser style would fit better with the beets style: something like:

    found 9 bad paths, can migrate 7
    not migrated: /some/path.mp3
    not migrated: /another/path.mp3

Or even:

    found 9 bad paths
    2 not migrated:
    /some/path.mp3
    /another/path.mp3